content/browser-automation.md

```diff
@@ -114,26 +114,20 @@ More information on [Selenium Webdriver](http://docs.seleniumhq.org/projects/web
 
 # Browser Automation Tools for Java
 
-Some browser automation tools are specific to Java.
-
 ## Serenity BDD
 
+{{% text "java" %}}
+
 Serenity BDD is an open source reporting library that helps you write better
 structured, more maintainable automated acceptance criteria. Serenity also produces
 rich meaningful test reports (or "living documentation") that report not only the
 test results, but also which features have been tested.
 
-{{% text "java" %}}
 A detailed tutorial on using Cucumber-JVM with Serenity can be found
 [here](http://thucydides.info/docs/articles/an-introduction-to-serenity-bdd-with-cucumber.html).
-{{% /text %}}
 
 The above scenario might be written for Serenity like this:
 
-```ruby
```

Comment:
This would be the alternate text for Ruby and javascript :) (see my previous comment)